### PR TITLE
[master][494337](UserStory) Add user-editable disclaimer to Python PDF script

### PR DESCRIPTION
**Pull request by arvidfm**
This pull request contains the following changes:

* Add disclaimer box to UI, the value of which will be printed in small print on each page
* Add clickable link to the generated file to the success alert
* Change fields for font size, rows, columns to be integer spinners instead of text fields

This pull request also incorporates changes from #12:

* When generating a PDF, the last used settings are saved to a settings file and used the next time the application is launched
* Move the Flix logo to a resource file that can easily be bundled with the application using pyinstaller
    * I have confirmed that the logo is read from the resource file correctly, but I was unable to create a working Linux binary using pyinstaller

### CODE REVIEW

File: pdf/pdf_ui.py [hunk 1]
```diff
@@ -148,6 +164,39 @@ def __create_line_label(self,
         label.setBuddy(line_edit)
         return line_edit, label
 
+    def __create_spinbox_label(self,
+                               value: int,
+                               label: str,
+                               minimum: int,
+                               maximum: int,
+                               min_width: int = 200,
+                               reg_exp: str = None) -> Tuple[Dict,
```
Comment (pscadding) [inline pdf/pdf_ui.py:173]: the `reg_exp` doesn't appear to be used?

File: pdf/pdf_ui.py [hunk 2]
```diff
@@ -148,6 +164,39 @@ def __create_line_label(self,
...
+                                                             Dict]:
+        """__create_line_label will create a line edit button and his label
```
Comment (pscadding) [inline pdf/pdf_ui.py:175]: his? 😛

**Review by pscadding — CHANGES_REQUESTED** [review]
just a few small comments

File: pdf/main.py [hunk 3]
```diff
@@ -36,6 +45,50 @@ def __init__(self, parent=None):
...
+            "export_path": self.wg_pdf_ui.export_path.text(),
+            "disclaimer": self.wg_pdf_ui.disclaimer.toPlainText(),
+        }
+
+        try:
+            SETTINGS_FILE.parent.mkdir(parents=True, exist_ok=True)
+            with open(SETTINGS_FILE, "w") as config_file:
+                yaml.safe_dump(curr_settings, config_file)
+        except OSError as e:
+            traceback.print_exc()
```
[the remark this answers is not in the document]
Comment (arvidfm) [reply pdf/main.py:89]: Apparently `QSettings` writes to the registry by default on Windows which sounds a bit scary and overkill for this application

[the remark this answers is not in the document]
Comment (arvidfm) [reply pdf/main.py]: And when using `IniFormat` typing information gets lost, which is unfortunate too... I'm kind of ambivalent on which approach to use, they both have pros and cons.

**Pull request merged**